internal/components/fluentd/run_script.go

```diff
@@ -132,6 +132,46 @@ do
     fi
 done
 
+FILE="/var/lib/fluentd/pos/journal_pos.json"
+
+    echo "$FILE exists."
+
+if test -f "$FILE"; then
+    echo "$FILE exists."
+    ruby -v /etc/fluent/configs.d/user/cleanInValidJson.rb  2> /tmp/error-json-parser.log
+
+    if [ -s /tmp/error-json-parser.log ]; then
+      echo "its invalid json"
+      sudo rm $FILE
```

Comment:
should we use `rm -f $FILE`
add a message here, saying which file is removed

Reply:
There is no sudo in the image, we can use the rm -f $FILE without sudo.